app/controllers/report_controller/dashboards.rb

```diff
@@ -51,6 +51,61 @@ def db_new
     db_edit
   end
 
+  def db_copy
```

Comment:
I suppose there is no API support for copy action right?

If not, can you please create a note to the forms sheet so we don't forget to add it afterwards?

Reply:
:+1: Yeah, after merging this I will update the sheet!

Reply:
@rvsia once we have API support for this, we can enhance this form to show the list of widgets that are part of Dashboard that's getting copied.